op-chain-ops/genesis/config.go

```diff
@@ -266,6 +266,19 @@ type DeployConfig struct {
 
 	// When Cancun activates. Relative to L1 genesis.
 	L1CancunTimeOffset *hexutil.Uint64 `json:"l1CancunTimeOffset,omitempty"`
+
```

Comment:
Can you comment in this file whether or not these config values are for superchain wide or just a single chain? This will help a lot in the future when we try to decouple the deploy config so that we chain chain specific config and then superchain specific config

Reply:
Also, would be great to link to our specs here which contain more details on all these params

Reply:
I've added that superchain comment. @mds1 I can't find any other locations in source code in the monorepo that link to the specs, IMO doing so could become hard to maintain.